store/kv/measure.go

```diff
@@ -118,14 +118,6 @@ func (m *TxImplWithMetrics) Delete(ctx context.Context, table []byte, key Key) (
 	return
 }
 
-func (m *TxImplWithMetrics) DeleteRange(ctx context.Context, table []byte, lKey Key, rKey Key) (err error) {
```

Comment:
We need to remove it from all other implementations of KV too.

Reply:
Thanks, done!